plugins/jobs/src/js/pages/JobRunHistoryTable.js

```diff
@@ -246,17 +248,34 @@ class JobRunHistoryTable extends React.Component {
   renderJobIDColumn(prop, row, rowOptions = {}) {
     if (!rowOptions.isParent) {
       const taskID = row.taskID;
-      const id = this.props.job.id;
+      // It may be that the data only *seems* to be present because we don't
+      // remove tasks from the MesosStateStore when they're garbage collected in
+      // the background.
+      const dataSeemsStillPresent = !!MesosStateStore.getTaskFromTaskID(taskID);
+
+      const link = (
+        <Link
+          className="table-cell-link-secondary"
+          to={`/jobs/detail/${this.props.job.id}/tasks/${taskID}`}
+          title={taskID}
+        >
+          <CollapsingString endLength={15} string={taskID} />
+        </Link>
+      );
+      const hint = (
+        <Tooltip
+          preferredDirections={["top-left", "bottom-left"]}
```

Comment:
What do you think about only having "bottom-left" here so that we don't hide the job id?
### Currently
![Screenshot from 2019-07-31 09-53-22](https://user-images.githubusercontent.com/40791275/62190148-529dd180-b379-11e9-9029-afe8b911bc39.png)

### My suggestion
![Screenshot from 2019-07-31 09-54-04](https://user-images.githubusercontent.com/40791275/62190155-57628580-b379-11e9-98a5-dd323bd6741f.png)

Reply:
really good catch!